### DISCUSSION TITLE
Serious!! can not see the info.

### DISCUSSION THREAD

**Discussion by yudalang3** [s-u/rJava#336] [Q&A] [answered]
Where is the error message in rJava package: This is my java code:

	public String launch() throws Exception {
//		Launcher.main(null);
//		Launcher4Dev.main(null);

		System.err.println("Error:" + new File("").getAbsolutePath());
		System.out.println("Output:" + new File("").getAbsolutePath());

		new InputMismatchException("fejfoejfwef").printStackTrace();

		return "Hello this is eGPS for R!";
	}
But I run it in the R:

  setLib_and_launchJVM();
  tryCatch(
    expr = {
      instance <- .jnew(launchClass)
      words <- .jcall(obj = instance, returnSig = "S", method = "launch")
      cat('The words of eGPS Java is: ', words, '\n');

    }
    ,
    Exception = function(e) {
      e$printStackTrace()
    }
  )
I can not see any error messages????

**Comment by s-u** [comment, marked as answer]
Since you cross-posted - see https://github.com/s-u/rJava/issues/335

**Comment by yudalang3** [comment]
I also give some solutions for using R in Rstuido:
```R
    temp.out <- tempfile(fileext = ".txt")
    on.exit(unlink(temp.out))
    jSys <- rJava::J("java/lang/System")
    jOrigOut <- jSys$out
    jSys$setOut(rJava::.jnew("java/io/PrintStream", temp.out))
    bioinfojavautils$go(rJava::.jarray(strsplit(cmd, "\\s+")[[1]]))
    jSys$setOut(jOrigOut)
```

This is a way
